src/vm/eetwain.cpp

```diff
@@ -3805,6 +3806,24 @@ bool UnwindEbpDoubleAlignFrame(
     {
         TADDR baseSP;
 
+#ifdef WIN64EXCEPTIONS
+        if (pCodeInfo->IsFunclet())
+        {
+            baseSP = curESP + 8;
```

Comment:
I'd also like to see a comment that describes what you expect the stack frame to look like, since you obviously have certain expectations about the stack layout.

---
In reply to: [101234990](https://github.com/dotnet/coreclr/pull/9601#discussion_r101234990) [](ancestors = 101234990)